src/BenchmarksClient/Startup.cs

```diff
@@ -102,64 +103,39 @@ public static int Main(string[] args)
 
         private static async Task ProcessJobs(CancellationToken cancellationToken)
         {
-            Process process = null;
+            IWorker worker = null;
 
             while (!cancellationToken.IsCancellationRequested)
             {
                 var allJobs = _jobs.GetAll();
                 var job = allJobs.FirstOrDefault();
                 if (job != null)
                 {
-                    var jobLogText =
-                        $"[ID:{job.Id} Connections:{job.Connections} Threads:{job.Threads} Duration:{job.Duration} Method:{job.Method} ServerUrl:{job.ServerBenchmarkUri}";
-
-                    Debug.Assert(job.PipelineDepth <= 0 || job.ScriptName != null, "A script name must be present when the pipeline depth is larger than 0.");
-
-                    if (!string.IsNullOrEmpty(job.ScriptName))
```

Comment:
These aren't needed for any scripts we're running still?

Reply:
The currently deployed clients have this code. When I deploy the new code it will use the properties.